src/Kunstmaan/NodeSearchBundle/EventListener/NodeIndexUpdateEventListener.php

```diff
@@ -44,7 +44,9 @@ public function onPostPersist(NodeEvent $event)
     private function index(NodeEvent $event)
     {
         $nodeSearchConfiguration = $this->container->get('kunstmaan_node_search.search_configuration.node');
-        $nodeSearchConfiguration->indexNodeTranslation($event->getNodeTranslation(), true);
+        $nodeTranslation = $event->getNodeTranslation();
+        $nodeSearchConfiguration->indexNodeTranslation($nodeTranslation, true);
+        $nodeSearchConfiguration->indexChildren($event->getNode(), $nodeTranslation->getLang());
```

Comment:
Disregard my last. It’s on an event either way, so any additional logic can be attached in client bundles.